test/api/handlers_test.go

```diff
@@ -133,6 +133,11 @@ func testHandlerUserPOST(t *testing.T, at *test.AccountsTester) {
 	if err != nil {
 		t.Fatal("Error while fetching the user from the DB. Error ", err.Error())
 	}
+	// Make sure the creation timestamp is correct.
+	now := time.Now().UTC()
+	if u.CreatedAt.Before(now.Add(-1*time.Minute)) || u.CreatedAt.After(now.Add(time.Minute)) {
```

Comment:
nit: would it not be sufficient to just verify that the timestamp is not zero?

Reply:
For this particular issue - yes, I can just compare to the zero value. But since we're adding a check anyway, we might as well make sure the time is accurate. This would catch, for example, an omitted `.UTC()` call and other things like that.

Reply:
`nit` could use `Truncate` maybe 

```go
if u.CreatedAt.Truncate(time.Minute) != now.Truncate(time.Minute)
```

Reply:
> `nit` could use `Truncate` maybe
> 
> ```go
> if u.CreatedAt.Truncate(time.Minute) != now.Truncate(time.Minute)
> ```

That opens us up to NDFs. `Truncate` doesn't round, it just drops the rest of the time, so 12:10:59.99 will truncate to 12:10 and 12:11:00.01 will truncate to 12:11.